static/examples/batch/batch-decomposition.scd

```diff
@@ -0,0 +1,42 @@
+(
+s.waitForBoot{
+    Task{
```

Comment:
We should be consistent between using `fork` / `Task` / <whatever> between examples

Reply:
Easy fix, thanks for pointing out. There are a few places where I should make the style consistent too. I have a preference for ({}) whenever the function is not anonymous. So basically everywhere except `someThing(action:{})`